appveyor.yml

```diff
@@ -3,11 +3,16 @@ max_jobs: 1
 image: Visual Studio 2017
 configuration: Release
 install:
+  - "C:\\Cygwin64\\setup-x86_64.exe -q -g -P libmpfr-devel,libgmp-devel"
+  - PATH c:\\Cygwin64\\bin;c:\\Cygwin64\\usr\\bin;%PATH%
+  - "C:\\Cygwin64\\bin\\bash -c '/usr/bin/make --directory=/cygdrive/c/projects/sleef/src/libm-tester -f Makefile.tester'"
```

Comment:
Ah - why do you need a makefile here? I am surprised the Cygwin cmake build does not work. The whole point of cmake is to avoid such craft around the builds.